lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -148,6 +148,15 @@ def __init__(self, namespace: str):
         self.client = Client()
         self.namespace = namespace
 
+    def delete_pod(self, pod_name: str) -> None:
+        """Deleting given pod.
+
+        Args:
+            pod_name    (str): Pod name to delete
```

Comment:
Why all the spaces between the argument and the type?

Reply:
Done